src/Common/Conjugation.fs

```diff
@@ -0,0 +1,32 @@
+module Conjugation
```

Comment:
A general point is that in theory this module is probably not supposed to live in the `Common` project which was meant to be a set of context-unrelated modules. It would be nice if we can rearrange dependencies and keep `Conjugation` in the `Core`->`Verbs` folder.